executors/async_scope/rename_join_nest.html

```diff
@@ -0,0 +1,688 @@
+<span class="citation" data-cites="P2300R10"><a href="https://wg21.link/p2300r10" role="doc-biblioref">[P2300R10]</a></span> is
+<code class="sourceCode cpp">let_value</code>. It, basically, runs the
+returned sender from the passed function and waits for its completion to
+send the computed values further. This
+returned-by-the-passed-function-sender is a nested work. <span class="citation" data-cites="P3149R9"><a href="https://wg21.link/p3149r9" role="doc-biblioref">[P3149R9]</a></span> itself contains the example
+with nested parallelism when shows handling a tree with
+<code class="sourceCode cpp">spawn_future</code>.</p>
+<p>However, <code class="sourceCode cpp">nest</code> name itself doesn’t
+have imply nested parallelism at all. That’s why it’s confusing. When
+users call <code class="sourceCode cpp">nest</code> they basically want
+to <code class="sourceCode cpp">associate</code> the piece of work with
+the scope, or <code class="sourceCode cpp">attach</code> the piece of
+work to the scope. The execution of the work is deferred. It might be
+also called like that by historical reasons: previously <span class="citation" data-cites="P3149R9"><a href="https://wg21.link/p3149r9" role="doc-biblioref">[P3149R9]</a></span> proposal had
+<code class="sourceCode cpp">nest</code> as one of the required
+operations on scopes by <code class="sourceCode cpp">async_scope</code>
+concept. The current state is that the
+<code class="sourceCode cpp">async_scope</code> concept was replaced by
+lower level one named
+<code class="sourceCode cpp">async_scope_token</code> (hopefully, just
+<code class="sourceCode cpp">scope_token</code> after applying <span class="citation" data-cites="P3685R0"><a href="https://isocpp.org/files/papers/P3685R0.pdf" role="doc-biblioref">[P3685R0]</a></span>) with requiring
+<code class="sourceCode cpp">try_associate</code> and
+<code class="sourceCode cpp">deassociate</code> methods (plus
+<code class="sourceCode cpp">wrap</code>). Also v1
+<code class="sourceCode cpp">async_scope</code> in <span class="citation" data-cites="libunifex"><a href="https://github.com/facebookexperimental/libunifex" role="doc-biblioref">[libunifex]</a></span> had the
+<code class="sourceCode cpp">attach</code> name instead of
+<code class="sourceCode cpp">nest</code>.</p>
+<p>In our desire to avoid confusion and also taking into account the
+previous experience we can recommend two names instead of
+<code class="sourceCode cpp">nest</code>. It’s either
+<code class="sourceCode cpp">associate</code> or
+<code class="sourceCode cpp">attach</code>. Given that <span class="citation" data-cites="P3149R9"><a href="https://wg21.link/p3149r9" role="doc-biblioref">[P3149R9]</a></span> talks about association a lot,
+this is the preferred option, unless authors of <span class="citation" data-cites="P3149R9"><a href="https://wg21.link/p3149r9" role="doc-biblioref">[P3149R9]</a></span> the similarity with
```

Comment:
This seems to be missing a verb. "Unless the authors of P3149 the similarity with...", should perhaps be something like "Unless the authors of P3149 disagree (?), the similary with..." ?